page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/capabilities/SelenoidCapabilitiesParser.java

```diff
@@ -46,6 +44,14 @@ public DesiredCapabilities parse() {
         return capabilities;
     }
 
+    private String getProperty(String property) {
+        if (property.trim().isEmpty() && Environment.getScenario() != null) {
```

Comment:
Do not return twice

return property.trim().isEmpty() && Environment.getScenario() != null
  ? Environment.getScenario().getName()
  : property;